earth2mip/networks/graphcast.py

```diff
@@ -135,6 +135,43 @@ def get_channel_names(variables, pressure_levels):
     return names
 
 
+def _get_array_type(x):
```

Comment:
Might be better to rename this to something a bit more descriptive? `_get_numpy_proxy` maybe?

Reply:
renaming to `get_array_module` which has a similar counterpart in cupy.